sdk/storage/Azure.Storage.Files.DataLake/tests/DirectoryClientTests.cs

```diff
@@ -876,7 +876,8 @@ public async Task SetAccessControlAsync()
         public async Task SetAccessControlAsync_RootDirectory()
         {
             await using DisposingFileSystem test = await GetNewFileSystem();
-            DataLakeDirectoryClient directory = InstrumentClient(test.FileSystem.GetRootDirectoryClient());
+            //DataLakeDirectoryClient directory = InstrumentClient(test.FileSystem.GetRootDirectoryClient());
```

Comment:
I think we should just delete this. It will be strange to somebody who does not understand the history of this feature. And it's not clear when we will add this API back.